samples/net/mqtt/src/modules/transport/credentials_provision/credentials_provision.c

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <modem/modem_key_mgmt.h>
+
+static const unsigned char ca_certificate[] = {
+#include "ca-cert.pem"
+};
+
+static const unsigned char device_certificate[] = {
+#include "client-cert.pem"
+};
+
+static const unsigned char private_key[] = {
+#include "private-key.pem"
+};
+
+int credentials_provision(void)
+{
+	int err;
```

Comment:
this spawns a warning if all three files are empty, maybe use `int err = 0;`

Reply:
Jepp, that works.